### ISSUE TITLE
GG! best accuracy 0.000000 in Feature encoder pre-training stage

### ISSUE THREAD

**Issue by e96031413** [hytseng0509/CrossDomainFewShot#27]
**command:**
```
CUDA_VISIBLE_DEVICES=0 python3 train_baseline.py --method baseline++ --dataset cars --name baseline++ --train_aug
```
**output:**
```
Epoch 399 | Batch 75/257 | Loss 0.226424
Epoch 399 | Batch 100/257 | Loss 0.233635
Epoch 399 | Batch 125/257 | Loss 0.232594
Epoch 399 | Batch 150/257 | Loss 0.234371
Epoch 399 | Batch 175/257 | Loss 0.231336
Epoch 399 | Batch 200/257 | Loss 0.230566
Epoch 399 | Batch 225/257 | Loss 0.225214
Epoch 399 | Batch 250/257 | Loss 0.222642
GG! best accuracy 0.000000
```

No matter use PyTorch 1.4 or the latest version(PyTorch 1.7), I got the same problem.

**Comment by Wystan0608** [comment]
Have you solved it? I have same problem.

**Comment by e96031413** [comment]
@Almostlover0608 Hi, I found that the train_baseline code did not calculate the best accuracy during training
Thus, the output  ```GG! best accuracy 0.000000``` appears in the output.

**Comment by hragwort** [comment]
Have you solved it? I have same problem.
